website/docs/d/mobile_network_attached_data_network.html.markdown

```diff
@@ -0,0 +1,102 @@
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Mobile Network Attached Data Network. Must be as same as `azurerm_mobile_network_data_network`, Changing this forces a new Mobile Network Attached Data Network to be created.
+
+* `mobile_network_packet_core_data_plane_id` - (Required) Specifies the ID of the Mobile Network Attached Data Network. Changing this forces a new Mobile Network Attached Data Network to be created.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Mobile Network Attached Data Network.
+
+* `location` - The Azure Region where the Mobile Network Attached Data Network should exist. 
+
+* `dns_addresses` - The DNS servers to signal to UEs to use for this attached data network.
+
+* `network_address_port_translation` - A `network_address_port_translation` block as defined below.
+
+* `user_plane_access_name` - The logical name for thie user data plane interface.
+
+* `user_plane_access_ipv4_address` - The IPv4 address for the user data plane interface.
+
+* `user_plane_access_ipv4_subnet` - The IPv4 subnet for the user data plane interface.
+
+* `user_plane_access_ipv4_gateway` - The default IPv4 gateway for the user data plane interface.
+
+* `user_equipment_address_pool_prefix` - The user equipment (UE) address pool prefixes for the attached data network from which the packet core instance will dynamically assign IP addresses to UEs. The packet core instance assigns an IP address to a UE when the UE sets up a PDU session.
+
+* `user_equipment_static_address_pool_prefix` - The user equipment (UE) address pool prefixes for the attached data network from which the packet core instance will assign static IP addresses to UEs. The packet core instance assigns an IP address to a UE when the UE sets up a PDU session. The static IP address for a specific UE is set in StaticIPConfiguration on the corresponding SIM resource.
+
+* `tags` - A mapping of tags which should be assigned to the Mobile Network Attached Data Network.
+
+---
+
+A `network_address_port_translation` block supports the following:
+
+* `pinhole_limits` - Maximum number of UDP and TCP pinholes that can be open simultaneously on the core interface. For 5G networks, this is the N6 interface. For 4G networks, this is the SGi interface.
+
+* `pinhole_timeouts_in_seconds` - A `pinhole_timeouts_in_seconds` block as defined below.
+
+* `icmp_pinhole_timeouts_in_seconds` - Pinhole timeout for ICMP pinholes in seconds. 
+
+* `tcp_pinhole_timeouts_in_seconds` - Pinhole timeout for TCP pinholes in seconds.
+
+* `udp_pinhole_timeouts_in_seconds` - Pinhole timeout for UDP pinholes in seconds.
+
+* `port_range` - A `port_range` block as defined below.
+
+* `port_reuse_minimum_hold_time_in_seconds` - A `port_reuse_minimum_hold_time_in_seconds` block as defined below.
```

Comment:
Again, this doesn't exist in the schema, left over from flattening the schema to the two properties below?